scripts/add_existing_baseyear.py

```diff
@@ -660,6 +710,11 @@ def add_heating_capacities_installed_before_baseyear(
 
     if options["heating"]:
 
+        # one could use baseyear here instead (but dangerous if no data)
+        heating_efficiencies = (
+            pd.read_csv(snakemake.input.heating_efficiencies, index_col=[1, 0])
+        ).loc[int(snakemake.config["energy"]["energy_totals_year"])]
```

Comment:
`snakemake.config` should no longer be used in `scripts/*.py` files due to new scenario management.

```suggestion
        fn = snakemake.input.heating_efficiencies
        year = int(snakemake.params["energy_totals_year"])
        heating_efficiencies = pd.read_csv(fn, index_col=[1, 0]).loc[year]
```

And then add to `add_existing_baseyear` rule definition:

```
params:
    energy_totals_year=config_provider("energy", "energy_totals_year")
```